ecl/ClassificationTestModified.ecl

```diff
@@ -0,0 +1,95 @@
+﻿/*##############################################################################
+## HPCC SYSTEMS software Copyright (C) 2022 HPCC Systems®.  All rights reserved.
+############################################################################## */
+
+#ONWARNING(2007, ignore);
+
+//version tmod='LowDep'
+//version tmod='MidDep'
+//version tmod='HighDep'
+
+// Modified version of the testCovTypeClass test file that works with the
+// OBT test system
+
+IMPORT $.^.test.datasets.CovTypeDS;
+IMPORT $.^ AS LT;
+IMPORT LT.LT_Types;
+IMPORT ML_Core;
+IMPORT ML_Core.Types;
+
+numTrees := 100;
+maxDepth := 255;
+numFeatures := 0; // Zero is automatic choice
+balanceClasses := FALSE;
+nonSequentialIds := TRUE; // True to renumber ids, numbers and work-items to test
+                            // support for non-sequentiality
+numWIs := 2;     // The number of independent work-items to create
+maxRecs := 5000; // Note that this has to be less than or equal to the number of records
+                 // in CovTypeDS (currently 5000)
+										
+t_Discrete := Types.t_Discrete;
+t_FieldReal := Types.t_FieldReal;
+DiscreteField := Types.DiscreteField;
+NumericField := Types.NumericField;
+trainDat := CovTypeDS.trainRecs;
+testDat := CovTypeDS.testRecs;
+ctRec := CovTypeDS.covTypeRec;
+nominalFields := CovTypeDS.nominalCols;
+numCols := CovTypeDS.numCols;
+
+ClassTest(INTEGER Dependentvar) := FUNCTION
+	ML_Core.ToField(trainDat, trainNF); // Get training data as a field
+	ML_Core.ToField(testDat, testNF); // Get test data as a field
+
+	//Ind = independent, Dep = dependent
+	Ind1 := PROJECT(trainNF(number != 52 AND id <= maxRecs), TRANSFORM(NumericField,
```

Comment:
We need to avoid the magic numbers like '52'. Please name it (store it a self documenting variable/constant).